R-package/R/lgb.Booster.R

```diff
@@ -335,7 +335,7 @@ Booster <- R6::R6Class(
 
       lower_bound <- 0.0
       lgb.call(
-        "LGBM_BoosterGetLowerBoundValue_R"
+        fun_name="LGBM_BoosterGetLowerBoundValue_R"
```

Comment:
```suggestion
        fun_name = "LGBM_BoosterGetLowerBoundValue_R"
```

Can you please put a space between the argument name and value for all of these cases?

Reply:
Okay..will be done soon